test/fixtures.py

```diff
@@ -0,0 +1,31 @@
+from shutil import copyfile
+
+import pytest
```

Comment:
Looks nicer when the imports are consistent.

For example start with 'import' imports and follow them with 'from' imports. And try to group the standard library imports and library imports separately when makes sense.

Reply:
Yes, I'm trying to do so. Just forgot to format the file correctly. Will fix it